app/labor/profile_image.rb

```diff
@@ -5,7 +5,7 @@ class ProfileImage
   def initialize(resource)
     @resource = resource
     @image_link = resource.profile_image_url
-    @backup_link = "https://thepracticaldev.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png"
+    @backup_link = "https://#{ApplicationConfig['AWS_BUCKET_NAME']}.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png"
```

Comment:
wouldn't this break local installations when merged? 
For example:

- the local `AWS_BUCKET_NAME` is not `thepracticaldev` by default, it's `Optional`
- mine is called `devto-tests` because I configured it to point to my AWS installation
- this would mean that I'd have to manually upload a file named `99mvlsfu5tfj9m7ku25d.png` for this to work

I think it'd have an impact on the other communities as well.

Maybe we could keep this hardcoded until we find a better way to handle this:

- either we download these images and put them in our assets (my fav option)
- or we find a way to upload them when the app is setup, through a script or something

Basically I don't think these images should be in AWS at all :)